.github/workflows/publish.yml

```diff
@@ -220,7 +220,7 @@ jobs:
           MESSAGE="$(cat ./releases/${VERSION}.md)"
           assets=()
           for asset in ./dist/*; do
-            assets+=("-a" "$asset")
+            assets+=("$asset")
           done
-          hub release create "${assets[@]}" -m "$VERSION" -m "$MESSAGE" "$VERSION"
-          
+          gh release create "$VERSION" --target "$VERSION" -F "./releases/${VERSION}.md" "${assets[@]}"
```

Comment:
Should we use `$VERSION` as `--target`, or `$GITHUB_SHA`?

I'm not sure if `--target` supports tags.

Reply:
I'm basically reproducing the current behavior. We now target a tag.